matlab/src/matlab/+arrow/+internal/+test/+io/+feather/roundtrip.m

```diff
@@ -1,6 +1,5 @@
-function tableOut = featherRoundTrip(filename, tableIn)
-% FEATHERROUNDTRIP Helper function for round tripping a table
-% to a Feather file.
+% ROUNDTRIP Test utility for round tripping a MATLAB
+% table to a Feather V1 file.
```

Comment:
to/from

Reply:
Thanks - I modified the wording. I think it is a bit more descriptive now.

Reply:
Marking this as resolved.